package.json

```diff
@@ -7,11 +7,15 @@
   "description": "A simple Node app built on Express, instantly up and running.",
   "main": "server.js",
   "scripts": {
-    "start": "node server.js"
+    "start": "node server.js",
+    "dev": "nodemon server"
   },
   "dependencies": {
```

Comment:
When you commit your code, please make sure that you don't commit the `node_modules` directory. As you can see there are over 700 files included in the pull request. All that you'd need to include in the pull request are `package.json`, `package-lock.json` and `server.js`.

Reply:
:+1: